interface/src/App.tsx

```diff
@@ -91,13 +93,60 @@ export const App = () => {
                 }    
             }
         }
-    
+        
+        async function fetchFontFamily() {
```

Comment:
Write what possible values are for font type

Reply:
Agree with you